components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx

```diff
@@ -217,13 +217,22 @@ export default class SystemUsersDropdown extends React.PureComponent {
             for (const bot of Object.values(this.props.bots)) {
                 if (bot.owner_id === user.id) {
                     messageForUsersWithBotAccounts = (
-                        <FormattedMarkdownMessage
-                            id='deactivate_member_modal.desc.for_users_with_bot_accounts'
-                            defaultMessage='This action deactivates {username}.\n \n * They will be logged out and not have access to any teams or channels on this system.\n * Bot accounts they manage will be disabled along with their integrations. To enable them again, go to Integrations > Bot Accounts. [Learn more about bot accounts](!https://mattermost.com/pl/default-bot-accounts).\n \n \n'
-                            values={{
-                                username: user.username,
-                            }}
-                        />);
+                        <p>
+                            <p>
+                                <FormattedMarkdownMessage
+                                    id='deactivate_member_modal.desc.for_users_with_bot_accounts'
+                                    defaultMessage='This action deactivates {username}.'
+                                    values={{
+                                        username: user.username,
+                                    }}
+                                />
+                            </p>
+                            <FormattedMarkdownMessage
+                                id='deactivate_member_modal.desc.for_users_with_bot_accounts_info'
+                                defaultMessage='\n * They will be logged out and not have access to any teams or channels on this system.\n * Bot accounts they manage will be disabled along with their integrations. To enable them again, go to Integrations > Bot Accounts. [Learn more about bot accounts](!https://mattermost.com/pl/default-bot-accounts).'
```

Comment:
@asaadmahmood Is the `\n * ` part intentionally part of the string? Wondering if that should instead handled by the JSX code itself

Reply:
Yes, that's intentional. This is a markdown formatted message.